src/brpc/socket.cpp

```diff
@@ -70,6 +70,10 @@ DEFINE_bool(log_idle_connection_close, false,
             "Print log when an idle connection is closed");
 BRPC_VALIDATE_GFLAG(log_idle_connection_close, PassValidate);
 
+DEFINE_int32(socket_server_tcp_keepalive, true, 
```

Comment:
This name is inconsistent with the impl. which works for client as well.

Reply:
Thank you for review.
I will change the name to `socket_enable_tcp_keepalive`